components/CardList.js

```diff
@@ -3,56 +3,43 @@ import styled from "styled-components";
 import MultiSelectDropdown from "./StyledElements/MultiSelect";
 import SvgIcon from "./StyledElements/SvgIcon";
 
-const StyledListContainer = styled.div`
+const PageContainer = styled.div`
   display: flex;
-  flex-wrap: wrap;
-  gap: 1rem;
+  flex-direction: column;
   align-items: center;
 `;
 
-const StyledList = styled.ul`
-  list-style: none;
+const ListContainer = styled.div`
   display: flex;
-  flex-wrap: wrap;
-  padding: 0;
-  margin: 0;
-  width: 100%;
-  gap: 1rem;
+  flex-direction: column;
+  align-items: center;
 `;
 
-const StyledListElement = styled.li`
-  flex: 1 1 calc(50% - 1rem);
-  display: flex;
-  flex-direction: column;
-  gap: 1rem;
-  max-width: 300px;
-  background-color: #fff;
-  border-radius: 8px;
-  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
-  overflow: hidden;
-  transition: transform 0.2s;
+const StyledList = styled.ul`
+  list-style: none;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
```

Comment:
Property grid-template-columns is overwritten

Reply:
I do not get this comment - what is the issue? ```Grid-template-columns```is overwritten with media-queries to adapt the grid based on different screen sizes - that is on purpose or did you mean something else?

Reply:
``` javascript
  grid-template-columns: repeat(4, 1fr);
  grid-template-columns: auto;
```
Doesn't the second line overwrite the first one?